main.cpp

```diff
@@ -3,6 +3,17 @@
 #include "ColorPair.h"
 namespace TelCoColorCoder
 {
+    std::string printColorCodingManual(){
```

Comment:
This function is together with test code in the same file. It means we cannot re-use it elsewhere.

Reply:
I've moved all the test to a new file and also moved the manual declaration to colorpair.cpp